App.js

```diff
@@ -84,125 +82,132 @@ i18n.fallbacks = true;
-  Settings: SettingsStackNavigator, // SettingsStack
-  Auth: AuthStackNavigator, // AuthStackNavigator
-}, { 
-  headerMode: 'none',
-  transitionConfig: (nav) => handleCustomTransition(nav),
-})
-
-const AppContainer = createAppContainer(MainStackNavigator)
+const MainStackNavigator = createStackNavigator(
+    {
+        Home: HomeScreen,
+        About: AboutScreen,
+        Flight: FlightStackNavigator, // FlightStack
+        NoFlight: NoFlightStackNavigator, // NoFlightStack
+        Payment: PaymentScreen,
+        ThankYou: ThankYouScreen,
+        UserProfile: UserProfileScreen,
+        Settings: SettingsStackNavigator, // SettingsStack
+        Auth: AuthStackNavigator, // AuthStackNavigator
+    },
+    {
+        headerMode: "none",
+        transitionConfig: (nav) => handleCustomTransition(nav),
+    }
+);
+
+const AppContainer = createAppContainer(MainStackNavigator);
 
 export default class App extends React.Component {
-  state = {
-    isLoadingComplete: false,
-  }
-
-  _loadResourcesAsync = async () => {
-    return Promise.all([
-      Asset.loadAsync([
-        require('./src/assets/background/home/home-1.png'), 
-        require('./src/assets/background/home/home-2.png'),
-        require('./src/assets/background/home/home-3.png'),
-        require('./src/assets/background/home/home-4.png'),
-        require('./src/assets/background/home/home-5.png'),
-      ]),
-      Font.loadAsync({
-        'Montserrat': require('./src/assets/fonts/Montserrat-Regular.ttf'),
-        'Montserrat-bold': require('./src/assets/fonts/Montserrat-Bold.ttf'),
-        'Fago-black': require('./src/assets/fonts/Fago-Black.ttf'),
-        'Gilroy-bold': require('./src/assets/fonts/Gilroy-Bold.ttf'),
-        'Poppins': require('./src/assets/fonts/Poppins-Medium.ttf')
-      })
-    ])
-  };
-
-  _handleLoadingError = error => {
-    console.warn(error);
-  }
-
-  _handleFinishLoading = () => {
-    this.setState({ isLoadingComplete: true });
-  }
-
-  render() {
-    if (!this.state.isLoadingComplete) {
-      return (
-        <AppLoading
-          startAsync={this._loadResourcesAsync}
-          onError={this._handleLoadingError}
-          onFinish={this._handleFinishLoading}
-        />
-      )
-    } else {
-      return(
-        <AppContainer/>
-      )
+    state = {
+        isLoadingComplete: false,
+    };
+
+    _loadResourcesAsync = async () => {
+        return Promise.all([
+            Asset.loadAsync([
+                require("./src/assets/background/home/bg_home.png"),
+            ]),
+            Font.loadAsync({
+                Montserrat: require("./src/assets/fonts/Montserrat-Regular.ttf"),
+                "Montserrat-bold": require("./src/assets/fonts/Montserrat-Bold.ttf"),
+                "Fago-black": require("./src/assets/fonts/Fago-Black.ttf"),
+                "Gilroy-bold": require("./src/assets/fonts/Gilroy-Bold.ttf"),
+                Poppins: require("./src/assets/fonts/Poppins-Medium.ttf"),
```

Comment:
should poppins and montserrat be in quotes?

Reply:
This is some weird react thing, it does it automatically